src/acf.py

```diff
@@ -122,12 +129,27 @@ def disconnect(self):
         except:
             pass
 
+    def timer_callback(self):
+        self.send_command(self.force)
+        self.initial_force = self.force
+        telem = self.handle_telem()
+        if self.joint_state_pub is not None:
+            msg = JointState()
+            msg.header.stamp = self.get_clock().now().to_msg()
+            msg.header.frame_id = f"{self.i}"
+            msg.name = [self.joint_name]
+            msg.position = [telem.position/1000]
+            msg.velocity = [0.0]    # TODO Calculate this by finite difference
+            msg.effort = [0.0]      # TODO Send the current force at the joint
```

Comment:
if you don't have this information right now, don't publish placeholders.

Exaggerating, but these lines are misrepresenting the device's state.

Reply:
I have removed the explicit setting of these values in [6989bbf](https://github.com/sam-xl/ferrobotics_acf/pull/2/commits/6989bbfe18451b0168b58bb63db34b602eafa8e5)